neqo-server/src/main.rs

```diff
@@ -836,6 +836,15 @@ fn main() -> Result<(), io::Error> {
     init_db(args.db.clone());
 
     if let Some(testcase) = args.qns_test.as_ref() {
+        if args.quic_parameters.quic_version.is_empty() {
+            // Quic Interop Runner expects the server to support `Version1` only.
+            // Exceptions are testcases `versionnegotiation` and `v2`. Neither are
+            // supported by Neqo. Thus always set `Version1`.
```

Comment:
Neqo does support v2?

Reply:
As far as I can tell, it does not implement the `v2` Quic Interop Runner testcase:

https://github.com/mozilla/neqo/blob/64fb41f47cbbbb1101484dcd37533c7b295fa659/neqo-client/src/main.rs#L989-L1017

Am I missing something @larseggert?

Reply:
Happy to work on it once the current test failures (https://github.com/quic-interop/quic-interop-runner/pull/344) are resolved.